FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/runspec_section.cpp

```diff
@@ -42,7 +42,8 @@ Runspec::Runspec(QStringList *driver_file_contents, QList<Model::Wells::Well *>
 
     // Create proper WELLDIMS keyword
     int maxwells = wells->size();
-    int maxconns = wells->first()->trajectory()->GetWellBlocks()->size() * (maxwells +1);
+//    int maxconns = wells->first()->trajectory()->GetWellBlocks()->size() * (maxwells +1);
+    int maxconns = 300;
```

Comment:
Why this change? Could it not cause problems?

Reply:
No problems. Old maxconns is sometimes too small, thus stopping the whole sim. Sometimes opt starts with really small wells that develop into very large ones.